icecream/icecream.py

```diff
@@ -154,15 +156,39 @@ def format_pair(prefix, arg, value):
     return '\n'.join(lines)
 
 
-def argumentToString(obj):
-    s = DEFAULT_ARG_TO_STRING_FUNCTION(obj)
+def argumentToString(obj, width=DEFAULT_LINE_WRAP_WIDTH):
+    s = DEFAULT_ARG_TO_STRING_FUNCTION(obj, width=width)
     s = s.replace('\\n', '\n')  # Preserve string newlines in output.
     return s
 
 
+def columns():
+    """ Returns the number of columns that this terminal can handle. """
+    width = DEFAULT_LINE_WRAP_WIDTH
+    try:
+        # TODO come up with a more elegant solution than subtracting
+        #  a seemingly random number of characters.
+        width = os.get_terminal_size().columns - COLUMN_OVERFLOW
```

Comment:
The docs https://docs.python.org/3/library/os.html#os.get_terminal_size suggest using shutil (https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size) instead, and looking at that function I think that's a good idea.

Reply:
Done.